charts/server/templates/statefulset.yaml

```diff
@@ -106,9 +106,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.extraVolumes }}
       volumes:
-      {{- if .Values.extraVolumes }}
-        {{- toYaml .Values.extraVolumes | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
Is there a practical different using the `with` notation instead of the existing one?

Reply:
first you do not need to use the howl path again (here in toYaml), so it is easier to read.

secound, `volumes:` is not need without `extraVolumes`